copilot-ops/main.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package main
+
+import "github.com/redhat-et/copilot-ops/cmd"
```

Comment:
Does this work for you ok? I thought you will need to add the `copilot-ops` subdir to imports, meaning `import "github.com/redhat-et/copilot-ops/copilot-ops/cmd"` 

I think we can move the go module to the root of the project, it is going to be the main go module of this project. WDYT?

Reply:
@guymguym Moved the directories up & placed the command definitions under `pkg/cmd`.